apps/web/ui/partners/external-payouts-indicator.tsx

```
@@ -0,0 +1,40 @@
+          </code>{" "}
+          <a
+            href={`/${slug}/settings/webhooks`}
+            target="_blank"
+            rel="noopener noreferrer"
+            className="cursor-alias underline decoration-dotted underline-offset-2"
+          >
+            webhook event.
+          </a>
+          <a
+            href="http://dub.co/docs/partners/external-payouts"
+            target="_blank"
```

Biome (2.1.2)

[error] Avoid using target="_blank" without rel="noopener" or rel="noreferrer".

Opening external links in new tabs without rel="noopener" is a security risk. See the explanation for more details.
Safe fix: Add the rel="noopener" attribute.


(lint/security/noBlankTarget)